src/compute/src/render/reduce.rs

```diff
@@ -1628,7 +1628,7 @@ where
                         if accum.is_negative() {
                             warn!("[customer-data] ReduceAccumulable observed a negative sum \
                                 accumulation over an unsigned type: {aggr:?}: {accum:?} in dataflow {debug_name}");
-                            soft_assert_or_log!(false, "Invalid negative unsigned aggregation in ReduceAccumulable");
+                            error!("Invalid negative unsigned aggregation in ReduceAccumulable");
```

Comment:
Does this make some of the `MZ_SOFT_ASSERTIONS=0` in the tests redundant now? If so, better to remove them, to not confuse readers.

Reply:
Absolutely, good catch!